pkg/config/app/kumactl/v1alpha1/config.proto

```diff
@@ -48,6 +52,9 @@ message ControlPlaneCoordinates {
     // ClientKey defines the key of the authorized client of the control plane
     // API server
     string client_key_file = 4;
+
+    // Headers to be added for communication with Kuma control plane
+    repeated AddHeader add_headers = 5;
```

Comment:
maybe just 
```
repeated Headers headers = 5;
```
and the argument `--header key=value`?

Reply:
ok